components/float-button/FloatButtonGroup.tsx

```diff
@@ -53,7 +54,9 @@ const FloatButtonGroup: React.FC<FloatButtonGroupProps> = (props) => {
 
   const mergedStyle: React.CSSProperties = { ...style, zIndex };
 
-  const wrapperCls = classNames(hashId, `${groupPrefixCls}-wrap`);
+  const wrapperCls = classNames(hashId, `${groupPrefixCls}-wrap`, {
```

Comment:
```suggestion
  const wrapperCls = classNames(hashId, groupPrefixCls, {
```

直接把所有的 -wrap 都去掉吧，感觉没什么用。

Reply:
breaking change 了，原有的逻辑就不动了吧，另外 `groupPrefixCls` 已经有外层的元素在用了

Reply:
这个 className 看着很难受，早晚要改，放 feature 改还合适点。

Reply:
那这里用 groupPrefixCls 的话，最外层的 groupPrefixCls 用什么？

Reply:
原来还有一个额外的包裹层，有可能打平么？

Reply:
应该可以无限内嵌 这个没啥问题，但是不管怎样 prefixCls-wrap 这层结构少不了，否则无法实现预期的布局

Reply:
另外这个和这次 PR 无关，这里就先不动了吧